core/payment-driver/dummy/src/service.rs

```diff
@@ -1,28 +1,34 @@
-use crate::processor::PaymentDriverProcessor;
+use ya_payment_driver::processor::PaymentDriverProcessor;
 use bigdecimal::BigDecimal;
 use ya_core_model::driver::*;
 use ya_persistence::executor::DbExecutor;
 use ya_service_bus::{typed as bus, RpcEndpoint};
 
+
+const BUS_ID_POSTFIX: &'static str = "dummy";
+
+
 pub fn bind_service(db: &DbExecutor, processor: PaymentDriverProcessor) {
     log::debug!("Binding payment driver service to service bus");
+    let bus_id: &str = &(BUS_ID_PREFIX.to_owned() + BUS_ID_POSTFIX);
```

Comment:
why are strings in rust so hard..

would like: ```
const BUS_ID: &'static str = BUS_ID_PREFIX + "dummy";
```

Reply:
You can use `lazy_static` to have it as `&'static str`.